manifests/params.pp

```diff
@@ -91,6 +91,7 @@
   $conf_d                     = "${confdir}/conf.d"
   $config_file                = "${confdir}/unbound.conf"
   $control_enable             = false
+  $control_setup_path         = '/usr/sbin/unbound-control-setup'
```

Comment:
I think what @igalic is suggesting, is that do avoid having to add the full path in params, is to add a `path =>` paramater to the `exec` resource.  This way when the call is made to the command, puppet will look in the path for the binary.